yt/chyt/controller/internal/jupyt/controller.go

```diff
@@ -107,6 +136,22 @@ func (c *Controller) GetOpBriefAttributes(parsedSpeclet any) map[string]any {
 	return nil
 }
 
+func (c *Controller) appendConfigs(ctx context.Context, oplet *strawberry.Oplet, speclet *Speclet, filePaths *[]ypath.Rich) error {
+	ytServerConfig := jupytServerConfig{
```

Comment:
`ytServerConfig`/`ytServerConfigPath` still look like chyt-specific naming.
But it's a minor style issue, so we can merge the PR even if it isn't fixed

Reply:
I mean "jupyterServerConfig on YT" here (not jupyter kernel). Renamed.